### ISSUE TITLE
`repl` fails with index out of range

### ISSUE THREAD

**Issue by nobe4** [nobe4/gh-not#58]
**Describe the bug**

```
$ gh-not repl
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/nobe4/gh-not/internal/views/command.New(_, _, _)
        /home/runner/work/gh-not/gh-not/internal/views/command/command.go:44 +0x54c
github.com/nobe4/gh-not/internal/views/normal.New(_, {_, _, _}, {_, _}, _)
        /home/runner/work/gh-not/gh-not/internal/views/normal/normal.go:101 +0xca8
github.com/nobe4/gh-not/internal/cmd.runRepl(0x140001b4a00?, {0x10070a0ac?, 0x4?, 0x100709f9c?})
        /home/runner/work/gh-not/gh-not/internal/cmd/repl.go:30 +0x80
github.com/spf13/cobra.(*Command).execute(0x100c190e0, {0x100c87720, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0x840
github.com/spf13/cobra.(*Command).ExecuteC(0x100c193c0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/nobe4/gh-not/internal/cmd.Execute(...)
        /home/runner/work/gh-not/gh-not/internal/cmd/root.go:47
main.main()
        /home/runner/work/gh-not/gh-not/cmd/gh-not/main.go:11 +0x24
```

**Version**
```
gh-not version v0.2.0 (16b4824e8ff0064f14ae0fa2f83c4602b0603f7a) built at 2024-06-15T11:31:31Z
https://github.com/nobe4/gh-not/releases/tag/v0.2.0
```

**Comment by nobe4** [comment]
Same stacktrace with:

```
go run cmd/gh-not/main.go repl
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/nobe4/gh-not/internal/views/command.New(_, _, _)
        /Users/nobe4/dev/nobe4/gh-not/internal/views/command/command.go:44 +0x54c
...
```

**Comment by nobe4** [comment]
The `command mode` mapping was missing from the keymap.

This is a good time to make a better arrangement for the configuration.

**Comment by nobe4** [comment]
Fixed by #60
Spawned #59
